### ISSUE TITLE
How to make it work with NSURLConnection

### ISSUE THREAD

**Issue by gdubs** [square/PonyDebugger#69]
So I've been trying to make it work with my app that uses NSURLConnection to call my APIs, and it's just now (after 3 days and comparing the PDTwitter example) that I realized the example project uses AFnetworking and that kinda affects if the requests get detected. 

I basically tried changing (as a test) the one I used from NSURLconnection to ASJSonRequestOperation. Before it would see my app on the listener but will not list anything on the Network tab, now it would show details regarding the request.

Basically my question is, I'm not sure if the NSURLConnection that I used was wrong that's why it wasn't detected or if PonyDebugger just chooses to detect AFNetworking methods only? 

I hope my question makes sense. 

Thanks!

**Comment by NickEntin** [comment]
We're doing some housecleaning, so I'm going to close this issue since a solution has been proposed.  Please reopen if this is still a problem.  Thanks.
